library/cwm/src/lib/cwm/page.rb

```diff
@@ -0,0 +1,53 @@
+module CWM
+  # A page widget is a group of widgets that are contained within it.
+  # Several pages are in turn contained within a {Pager}.
+  #
+  # {TreePager} is a {Pager}.
+  #
+  # {Tabs} is a {Pager} and a {Tab} is its {Page}
+  # (FIXME: We haven't made Tabs a subclass of Pager yet.
+  # That will come after we have TreePager working.
+  class Page < CustomWidget
+    # @return [Boolean] is this the initially selected tab
+    attr_accessor :initial
+
+    # @return [WidgetTerm] contents of the tab, can contain {AbstractWidget}s
+    attr_reader :contents
+    # @return [String] label defines name of tab header
```

Comment:
??? this comment mention tab header, but this is generic Page and as mentioned above Tab is Page, but not vice verse. So I do not see explained it anywhere in comment.